gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
This is a gradle newbie question: what is the advantage of having all these gradlew scripts committed to all the different repositories that use gradle. Isn't this copy-paste a maintenance nightmare?

Reply:
This is the recommend way to use Gradle. The two main advantages are that people building the projects don't have to install Gradle and it insures that everyone building the project uses the same version of Gradle. (More info https://docs.gradle.org/current/userguide/gradle_wrapper.html)

I'm not sure what you mean by a copy-paste nightmare, but these files are generated.  For example to upgrade the version of Gradle, you run  `./gradlew wrapper --gradle-version 4.2.1`

Reply:
Do we need different scripts in checker-framework and checker-framework inference?
Wouldn't it be nicer to have one consistent version that the CF and all dependent projects use?

Reply:
The convention is to commit the scripts and jar files to each project.  (So that each one can be built in a reproducible way.)
We could choose to always update one whenever updating the other.